frontend/src/components/app/app.tsx

```diff
@@ -36,7 +37,7 @@ const App: FC = () => {
     dispatch(appActions.getTenant({ id: user.tenantId }));
   }, [user, dispatch]);
 
-  if (!hasUser && hasToken) {
+  if (!hasUser && hasToken && authStatus !== DataStatus.REJECTED) {
```

Comment:
What happens if auth would return error on every request? Isn't it a infinite loop?